packages/popover2/src/tooltip2.tsx

```diff
@@ -103,24 +105,16 @@ export class Tooltip2<T> extends React.PureComponent<Tooltip2Props<T>> {
     }
 
     public reposition() {
-        if (this.popover != null) {
-            this.popover.reposition();
-        }
+        this.popoverRef.current?.reposition();
     }
 
     // any descendant ContextMenu2s may update this ctxState
     private renderPopover = (ctxState: Tooltip2ContextState) => {
         const { children, disabled, intent, popoverClassName, ...restProps } = this.props;
-        const classes = classNames(
-            Classes.TOOLTIP2,
-            { [CoreClasses.MINIMAL]: this.props.minimal },
```

Comment:
this line was intentionally removed. it's not necessary to apply this class here because Popover2 will do it anyway since we're passing through the `minimal` prop